### PR TITLE
Add border for markdown field name

### PR DESCRIPTION
**Pull request by smoghul**

### CODE REVIEW

**Comment by smoghul** [comment]
@Thrillberg  what do you think of this?
Screen Shot 2019-07-09 at 8 23 40 AM

File: src/DocField.js [hunk 1]
```diff
@@ -4,13 +4,15 @@ class DocField extends Component {
   render() {
     const prefix = this.props.repoName + " / doc / goals /"
     const filepath =  (
+      <section className="docfield">
       <div>
```
Comment (Thrillberg) [inline src/DocField.js]: We should likely indent the `<div>` inside the `<section>`. What do you think?

**Review by Thrillberg — COMMENTED** [review]
Looks good, just one small indentation issue, and also I think there is a typo in "border" in the commit message.

File: src/DocField.js (hunk 1 above)
Comment (smoghul) [reply src/DocField.js]: Done!

**Review by Thrillberg — APPROVED** [review]
Looks great! Let's merge it!

**Pull request merged**